geest/core/workflows/default_index_score_workflow.py

```diff
@@ -29,6 +29,7 @@ def __init__(self, attributes: dict, feedback: QgsFeedback):
         :param feedback: QgsFeedback object for progress reporting and cancellation.
         """
         super().__init__(attributes, feedback)
+        self.layer_id = self.attributes["ID"].lower()
```

Comment:
@timlinux am I referencing the wrong layerid at this point

Reply:
no that all looks correct now thanks